packages/libs/react-ui/src/components/Stack/Stack.css.ts

```diff
@@ -0,0 +1,103 @@
+import { sprinkles, vars } from '../../styles';
+
+import { CSSProperties, style, styleVariants } from '@vanilla-extract/css';
+
+export const container = style([
+  sprinkles({
+    display: 'flex',
+    gap: 'md',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+    flexWrap: 'nowrap',
+  }),
+]);
+
+export const spacingClass = styleVariants(
+  {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    '2xs': '2xs',
+    xs: 'xs',
+    sm: 'sm',
+    md: 'md',
+    lg: 'lg',
+    xl: 'xl',
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    '2xl': '2xl',
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    '3xl': '3xl',
+  } as Record<string, keyof typeof vars.sizes>,
+  (gap) => {
+    return [
+      container,
+      sprinkles({
+        gap,
+      }),
+    ];
+  },
+);
+
+export const justifyContentClass = styleVariants(
+  {
+    'flex-start': 'flex-start',
+    center: 'center',
+    'flex-end': 'flex-end',
+    'space-between': 'space-between',
+    'space-around': 'space-around',
+  },
+  (justify) => {
+    return [
+      container,
+      {
+        justifyContent: justify,
+      },
+    ];
```

Comment:
Sprinkles seems to dislike being fed `justifyContent`, `alignItems`, `flexWrap` nor `flexDirection`. The last two also require the type to be defined, otherwise even without sprinkles ts complains :(